sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuExpressions.scala

```diff
@@ -46,6 +46,47 @@ object GpuExpressionsUtils extends Arm {
       throw new IllegalStateException("Internal Error GPU support for this data type is not " +
         "implemented and should have been disabled")
   }
+
+  /**
+    * Tries to resolve a `GpuColumnVector` from a Scala `Any`.
+    *
+    * This is a common handling of the result from the `columnarEval`.
+    * And it is recommended to return only one of the two types from a GPU expression's
```

Comment:
Most readers are going to ask, "which two types are you talking about?"

Reply:
Haha, Forgot to update it.
Now it is done.